lib/checks/keyboard/has-at-least-one-main.js

```diff
@@ -0,0 +1,3 @@
+var main = document.querySelector('main,[role=main]');
+this.data(!!main);
```

Comment:
This tells the after function whether or not the main exists; if there's a main in any one document, all of the documents will pass. This will probably make more sense when I add the file for the after function.